### PR TITLE
Remove duplicated text

### PR DESCRIPTION
**Pull request by fzipi**
Text seems repeated, removing.

**Checklist**:
- [x] updated docs
- [ ] unit tests added
- [ ] related issues referenced
- [ ] updated CHANGELOG.md
- [x] ready for review

### CODE REVIEW

File: docs/docs/readme.md [hunk 1]
```diff
@@ -36,7 +36,6 @@ Pomerium is composed of 3 logical components:
   - All user traffic flows through the proxy
   - Initiates authentication flow to Authentication service as needed
   - Verifies all requests with Authorization service
-  - Verifies all requests with Authorization service
```
[the remark this answers is not in the document]
Comment (travisgroth) [reply docs/docs/readme.md]: It probably was meant to be authorization

[the remark this answers is not in the document]
Comment (desimone) [reply docs/docs/readme.md]: @fzipi would you mind updating the text to authorization? Sorry for the confusion and thanks again for the PR.

[the remark this answers is not in the document]
Comment (fzipi) [reply docs/docs/readme.md]: @desimone Sure. You mean just changing case?

[the remark this answers is not in the document]
Comment (travisgroth) [reply docs/docs/readme.md]: Sorry guys.  I meant 'Authentication' service.  I made the same mistake twice :(

[the remark this answers is not in the document]
Comment (fzipi) [reply docs/docs/readme.md]: Excellent! Changed accordingly then :)

[the remark this answers is not in the document]
Comment (desimone) [reply docs/docs/readme.md]: Thank you @fzipi  :)

**Pull request merged**